src/server.rs

```diff
@@ -44,16 +42,29 @@ impl Server {
         let router = self.router.clone();
         let headers = self.headers.clone();
 
+        let asyncio = py.import("asyncio").unwrap();
+
+        let event_loop = asyncio.call_method0("new_event_loop").unwrap();
+        asyncio
+            .call_method1("set_event_loop", (event_loop,))
+            .unwrap();
+        let event_loop_hdl = PyObject::from(event_loop);
+
         thread::spawn(move || {
             //init_current_thread_once();
             actix_web::rt::System::new().block_on(async move {
                 let addr = format!("127.0.0.1:{}", port);
 
                 HttpServer::new(move || {
+                    let event_loop_hdl = event_loop_hdl.clone();
```

Comment:
Why not just move the value? (and not clone it)

Reply:
`HttpServer::new` expects a `Fn` not a `FnOnce`, so when `event_loop_hdl` gets moved into the router closure later on, `event_loop_hdl` needs to remain valid for the next time the `HttpServer::new` closure is called.

Reply:
Ah, didn't know that, then this LGTM 👍